test/util.go

```diff
@@ -1,10 +1,53 @@
 package test
 
 import (
+	"fmt"
+	"image"
+	"image/draw"
+	"image/png"
+	"os"
+	"path/filepath"
+	"testing"
+
 	"fyne.io/fyne"
 	"fyne.io/fyne/internal/cache"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+// AssertImagesEqual asserts that the master image at given filename (below testdata) and the given image are equal.
+// It fails the test if not. In this case the actual image is written to disk and reported.
+func AssertImagesEqual(t *testing.T, masterFilename string, got image.Image) {
```

Comment:
I think that this method name could be better - as a developer I would expect an equals check to operate on two of the same type. How about AssertImageMatchesMaster/Golden or something similar, though the order of the parameters would be reversed.